spec/syntax.md

```diff
@@ -452,6 +536,8 @@ The grammar defines the following tokens for the purpose of the lexical analysis
 
 ### Keywords
 
+A **_keyword_** is a reserved token that is part of the _message_ syntax.
```

Comment:
```suggestion
A **_keyword_** is a reserved token that has a unique meaning in the _message_ syntax.
```

(Technically many possible tokens are part of the _message_ syntax, not just keywords)